squarelet/templates/account/signup.html

```diff
@@ -136,6 +136,17 @@
             {{ form.password1.help_text }}
           </div>
         </label>
+        <label class="field">
+          <span class="label">
+          {{ form.tos }}
+          {{ form.tos.label }}
```

Comment:
This template already has some custom HTML in it already; I don't think we have to be too strict about using every field's programmed values:

```suggestion
          <span>I accept MuckRock&rsquo;s <a target="_blank" rel="noopener noreferer" href="https://www.muckrock.com/tos/">Terms of Service</a> and <a target="_blank" rel="noopener noreferer" href="https://www.muckrock.com/privacy-policy/">Privacy Policy</a>.</span>
```

With this, we can remove the help text and simplify the field down to one line.